uol-web-dev/index.html

```diff
@@ -1,15 +1,55 @@
-<!DOCTYPE html>
-<html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Work+Sans">
-    <title>MarvelDC Theme Park</title>
-  </head>
-  <body>
-
-  <script src="script.js" charset="utf-8"></script>
-  </body>
-</html>
```

Comment:
@SherminChai This code is the boilerplate, it should not be removed. Please _only_ add whatever code is within your `body` and `/body` tag into the original `body` tag!